wordpress-mobile/WordPress-Android
# QR Code Auth: Prep for release

Parent #16481 

This PR preps QR Code Auth flow for release
 - Enables the feature flag
 - Adds release notes
 
**To test:**
- Install the Jetpack and WordPress apps on your device
- Launch Jetpack
- Login using a WP.com account 
- Navigate to Me
- ✅ Verify `Scan Login Code` is visible in the list
- Launch WordPress 
- Login using a WP.com account 
- Navigate to Me
- ✅ Verify `Scan Login Code` is NOT visible in the list

## Regression Notes
1. Potential unintended areas of impact
The option shows when it shouldn't

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so) N/A

PR submission checklist:
